packages/phoenix-app/package.json

```diff
@@ -11,16 +11,17 @@
   },
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
-    "start:ssl": "ng serve --ssl",
+    "start": "ng serve --ssl --host=0.0.0.0",
```

Comment:
I think we don't need the ssl and host options in this. We mostly want to use that when testing VR so we can have the `start:ssl` script for that.

If we want the `phoenix-event-display` to start with the app with lerna, we can add a duplicate `start:ssl` script in `phoenix-event-display` which will just be `"start:ssl": "npm run start"`.

Reply:
Oops yes this is a mistake and shouldn't have been committed. Thanks!